modules/data.land/R/cohort2pool.R

```diff
@@ -8,14 +8,6 @@
 ##' @param dbh_name Default is "DBH". This is the column name in the veg_file that represents DBH. May differ depending on data source.
 ##' @param dat veg_info file
 ##' @param allom_param parameters for allometric equation, a and b. Based on base-10 log-log linear model (power law)
-##'
-##' @author Saloni Shah
-##' @examples
-##' \dontrun{
-##' veg_file <- "~/downloads/FFT_site_1-25665/FFT.2008.veg.rds"
-##' cohort2pool(veg_File = veg_file, allom_param = NULL)
-##' }
-##' 
```

Comment:
1. seems unrelated to current PR
2. Not sure why you are removing the example and deleting Saloni's authorship

Reply:
Yes, I am going to revert this commit